src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.cs

```diff
@@ -153,7 +153,14 @@ private DependencyPropertyChangedEventArgs CreateInitialValueChangerEventArgs(De
 		public string Text
 		{
 			get { return (string)this.GetValue(TextProperty); }
-			set { this.SetValue(TextProperty, value); }
+			set {
+				if (value == null)
+				{
+					throw new ArgumentNullException();
```

Comment:
Is it the behavior of UWP? Surprising...

Reply:
yep